platform-dom/src/test/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteDefaultInputEngineTest.java

```diff
@@ -30,7 +31,6 @@ public DomAutoCompleteDefaultInputEngineTest() {
 	@Test
 	public void testWithEmptyPattern() {
 
-		assertEquals("[]", toDisplayStrings(inputEngine.findMatches("", 0)));
```

Comment:
Because it seems pointless to start a search query which is limited to max 0 results. Why would we support that? The attempt will result in `IllegalArgumentException`, as of this PR.

Reply:
hmm, I don't like that, but ok